README

```diff
@@ -1 +1,3 @@
 README
+create new branch feature-D
```

Comment:
Why add this line ? :rabbit:

Reply:
Because this import for the user to know that this feature has bean in the sprint. :ribbon: